packages/r.rb

```diff
@@ -46,25 +45,7 @@ class R < Package
   depends_on 'xzutils' # R
   depends_on 'zlibpkg' # R
 
-  def self.build
-    @x = ARCH == 'i686' ? '' : '--with-x'
-    system "./configure #{CREW_OPTIONS} \
-           --disable-maintainer-mode \
+  configure_options "--disable-maintainer-mode \
```

Comment:
Maintainer mode is disabled by default in `4.3.2`.

https://github.com/wch/r-source/blob/tags/R-4-3-2/configure.ac#L321-L327